vm/interpreter/src/vm.rs

```diff
@@ -324,6 +324,29 @@ where
             fvm::executor::ApplyKind::Explicit,
             raw_length,
         )?;
+
+        let exit_code = ret.msg_receipt.exit_code;
+
+        if !exit_code.is_success() {
+            match exit_code.value() {
+                1..=11 => {
+                    log::debug!(
+                        "Internal message execution failure. Exit code was {}",
+                        exit_code
+                    )
+                }
+                16..=24 => {
```

Comment:
To my understanding of the code (I could be wrong), e.g. 12, 13, 26 or 27 are reserved codes, so the `ref-fvm` team may assign it even tomorrow. We would then need to know about it and bump the values.

I believe it'd be easier and more maintainable to have a simpler:
* if the exit code value is `1..FIRST_USER_EXIT_CODE` then it's an internal fiasco,
* else it's a fiasco in general

We wouldn't have to revisit it then in the future too often (unless a new category is created or something like that).

What do you think?

Reply:
I think this way was a lot simpler than matching over every possible exit code. You're right about the reserved codes but this just seemed more clean and practical given the FVM doesn't have a way to go from the exit code value to the name easily. I'll go ahead and change this up so we can provide more information to the user than just a number they'll have to look up

Reply:
Actually nevermind, changing up the match would be a lot simpler